### ISSUE TITLE
Update needed for Django 1.9+

### ISSUE THREAD

**Issue by adi-** [robpogorzelski/django-permissionsx#27]

**Comment by thinkingpotato** [comment]
I'm afraid this won't happen in any foreseeable future since I won't be involved in software development business for at least three years. However, I'm happy to give access to this repository or merge any changes. Keeping this open.

**Comment by thinkingpotato** [comment]
Thank you!

**Comment by adi-** [comment]
This one wasn't fix, yet. Changing/fixing import error doesn't help.

**Comment by thinkingpotato** [comment]
OK. I'll have a look into 1.9+ compatibility issues in a week or so, seems I'm back to Web programming for a while and the project requires permissions management of this kind.

**Comment by adi-** [comment]
Updating or closing?

**Comment by robpogorzelski** [comment]
I'm sorry, unfortunately there will be no updates to this package until it is "economically justified".

**Comment by robpogorzelski** [comment]
Back to business. What exactly is the issue? I'm running Py3.5 and Django 1.10 locally and can't reproduce any errors.

**Comment by adi-** [comment]
So far:

1. 

```
/permissionsx/contrib/django/views.py", line 11, in <module>
    from django.utils import importlib
ImportError: cannot import name 'importlib'
```
2. 

```
/permissionsx/utils.py", line 11, in <module>
    from django.utils import importlib
ImportError: cannot import name 'importlib'
```

**Comment by robpogorzelski** [comment]
So this one would be solved, at least if you're running Python 3.5+ since I plan on dropping support for Python 2.x.
